src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -412,6 +415,8 @@ public void setValues(JSONObject formData) {
             useRestApi = false;
         }
 
+        voteSameTopic = formData.optBoolean("voteSameTopic", false);
```

Comment:
why reading/writing the same value twice?

Reply:
Sorry, it's a mistake. Fixed.